src/group/errors.rs

```diff
@@ -44,3 +46,14 @@ pub enum ApplyCommitError {
 pub enum CreateCommitError {
     CannotRemoveSelf = 300,
 }
+
```

Comment:
Not a blocker, but we might want to log exact errors in the initial debugging phase before aggregating errors.

One way would be to have something like

```
WelcomeError::InvalidRatchetTree(TreeError)
```

Reply:
Yeah, this is a little basic and we should make error handling a little nicer.
I filed #83 to follow up on this.